index.bs

```diff
@@ -1100,6 +1100,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
             1. Return |constructCredentialAlg| and terminate this algorithm.
 
+        :   If the [=client=] concludes that it is impossible for any eligible [=authenticator=] to appear,
+        ::  The client MAY prompt the user for [=user consent|consent=] to [=authentication|authenticate=] to the [=[RP]=]. If
```

Comment:
hm, here we're adding this switch clause here to the [#createCredential](https://w3c.github.io/webauthn/#createCredential) alg, yes?

However, @christiaanbrand's issue #953 appears to be in regards to the [#getAssertion](https://w3c.github.io/webauthn/#discover-from-external-source) alg, even though he's not explicit about it, yes?

Reply:
Yes, I added it to both operations. The situation is relevant to both, but caused by different circumstances.

Reply:
Oh, but I had apparently swapped the Note:s between the two operations. That might not be what you asked about, but still, thanks!

Reply:
for some reason I saw the "If the client concludes that it is impossible for any eligible authenticator to appear" switch clause only in the [[Create]] alg, and not in [[DiscoverFrom...Source]].  cockpit error on my part.